src/py/aspen/config/config.py

```diff
@@ -230,6 +230,9 @@ def SECRET_KEY(self) -> str:
 
     @property
     def DATABASE_URI(self) -> str:
+        # Allow db uri to be overridden by env var.
+        if os.getenv("DB_URI"):
+            return os.getenv("DB_URI")
```

Comment:
I'm a little on the fence about adding yet another way to configure DB URI's here, but open to ideas.

Reply:
i think it's ok

Reply:
we could create like a factory method later on if we wanted to polish this later, but i think it's fine to add that in later PRs